src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,49 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, V> {
+    private T firstElement;
+    private V secondElement;
 
+    private Pair(T firstElement, V secondElement) {
+        this.firstElement = firstElement;
+        this.secondElement = secondElement;
+    }
+
+    public T getFirst() {
+        return firstElement;
+    }
+
+    public V getSecond() {
+        return secondElement;
+    }
+
+    public static <T, V> Pair of(T t, V v) {
+        return new Pair(t, v);
+    }
```

Comment:
```suggestion
    public static <T, V> Pair<T, V> of(T t, V v) {
        return new Pair<>(t, v);
    }
```
we should parametrize the return type too